docs/DeveloperGuide.md

```diff
@@ -2,28 +2,206 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+1. Reference to AB-3 Developer Guide
+
+- [Source](https://se-education.org/addressbook-level3/DeveloperGuide.html#proposed-undoredo-feature)
+- Used as template to structure this Developer Guide
 
```

Comment:
Shouldn't there be a table of contents to give readers easy access to various parts of the developer guide? Maybe putting one can help with the navigability of the guide!

Reply:
I agree!